tests/integration/types/sqlite.py

```
@@ -158,6 +159,17 @@ class FruitTypeWithPaginationAndOrderBy: ...
 class FruitFilter: ...
 
 
+def _fruit_sweeter_than(statement: Select[tuple[Fruit]], value: int, **_ctx: Any) -> Select[tuple[Fruit]]:
```

Ruff (0.15.17)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**_ctx`

(ANN401)